examples/rails/Gemfile

```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.6'
-# Use sqlite3 as the database for Active Record
+# Use postgres as the database for Active Record
+gem 'pg'
 gem 'sqlite3'
```

Comment:
config/database.yml references sqlite, so tools like `rails g model` complain without the Gem.

Could also update the config